iocore/dns/SRV.h

```diff
@@ -52,12 +52,12 @@ operator<(const SRV &left, const SRV &right)
 }
 
 struct SRVHosts {
-  unsigned srv_host_count;
-  unsigned srv_hosts_length;
+  unsigned int srv_host_count   = 0;
+  unsigned int srv_hosts_length = 0;
   SRV hosts[HOST_DB_MAX_ROUND_ROBIN_INFO];
 
   ~SRVHosts() {}
-  SRVHosts() : srv_host_count(0), srv_hosts_length(0) {}
+  SRVHosts() {}
```

Comment:
What I've been doing when we get empty default ctor / dtor's like this, is to just nuke them. They have no value any more afaik.